.travis.yml

```diff
@@ -43,7 +43,7 @@ script:
       # run dialyzer only in otp 22
       rebar3 dialyzer
       # test against kafka 0.9 in otp 22
-      export KAFKA_VERSION="0.9"
+      export KAFKA_VERSION="1.0"
```

Comment:
I was not clear enough.
kafka 0.9 docker image is no longer updated
but that does not imply we do not support 0.9 any more in `brod`

we can instead check kafka version (by e.g. query api version) in test case,
and simply *pass* it if it's not 1.0 or later.